common/hash.c

```diff
@@ -35,6 +35,141 @@
 #include <u-boot/sha512.h>
 #include <u-boot/md5.h>
 
+#if CONFIG_IS_ENABLED(MBEDTLS_LIB_CRYPTO)
+
+static int hash_init_sha1(struct hash_algo *algo, void **ctxp)
```

Comment:
I think we now require comments on all fucntions. We will have to check that

Reply:
I guess we don't need comments for the static one?